less/index.less

```diff
@@ -1,7 +1,32 @@
 // Import your less files here:
 
-
-
+@import "variables.less";
+@import "mixins.less";
+@import "reset.less";
+@import "global.less";
+@import "navigation.less";
+@import "footer.less";
+@import "home-page.less";
+
+
+
+
+@media @phone {
```

Comment:
We went over this in our 1:1 meeting but remember to close all your brackets (most of the less compiler errors we saw were from unclosed brackets) and move this to home-page.less